content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -4247,6 +4247,353 @@ const posts = await prisma.post.findMany({
 })
 ```
 
+## Composite type methods
+
+<Admonition type="warning">
+
+Available for MongoDB only in Prisma 3.12.0 and later.
+
+</Admonition>
+
+Composite type methods allow you to create, update and delete [composite types](/concepts/components/prisma-client/composite-types).
+
+### Reference
+
```

Comment:
I'm missing a type signature here

Reply:
Not sure where to find this, can someone point me to it?

Reply:
Via @millsp 'There aren't any composite type methods, since they are always embedded in a model, so they are always created, updated, deleted via their parent model.' So I'll remove this section.